docs/advanced_reference/commands/autonomy_mint.md

```diff
@@ -0,0 +1,126 @@
+# `autonomy mint`
+
+Tools to mint components on chain
+
+### Options
```

Comment:
If these are command-group options, it should be a 2nd level title.
## Command group options